mulighetsrommet-api/src/main/resources/db/migration/V242__to_trinnskontroll.sql

```diff
@@ -0,0 +1,43 @@
+drop view if exists tilsagn_admin_dto_view;
+drop view if exists tilsagn_arrangorflate_view;
+
+alter table tilsagn
+    drop column status_endret_av,
+    drop column status_besluttet_av,
+    drop column status_endret_tidspunkt,
+    drop column status_aarsaker,
+    drop column status_forklaring;
+
+create type to_trinnskontroll_type as enum (
+    'OPPRETT',
+    'ANNULLER'
+);
+
+create table to_trinnskontroll
+(
+    id                  serial primary key,
```

Comment:
Kan endres til `id int generated always as identity` (da dette følger standarden for identity colums), står litt mer utdypet her (tilfeldig blogg-post fra et google-søk): https://www.naiyerasif.com/post/2024/09/04/stop-using-serial-in-postgres/

Reply:
Today i learned :)